.github/workflows/release-crates.yml

```diff
@@ -0,0 +1,48 @@
+name: Release Crates
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        type: boolean
+        description: "DryRun release"
+        required: true
+        default: false
+
+permissions:
```

Comment:
Do we need these two permission while publish crates?

Reply:
Nice catch. I've removed.